pilot/pkg/model/jwks_resolver.go

```diff
@@ -142,10 +144,11 @@ func init() {
 }
 
 // NewJwksResolver creates new instance of JwksResolver.
-func NewJwksResolver(evictionDuration, refreshInterval, retryInterval time.Duration) *JwksResolver {
+func NewJwksResolver(evictionDuration, refreshDefaultInterval, refreshIntervalOnFailure, retryInterval time.Duration) *JwksResolver {
```

Comment:
As you're already here, could you rename this to newJwksResolver()? I think it's not used anywhere out of jwks_resolver.go and jwks_resolver_test.go. Thanks.